scripts/run_device_container.sh

```diff
@@ -8,6 +8,43 @@
 docker pull huntsmanarray/device_startup:latest
 
 #Run the docker container
+
 #Note that network=host to map container's ports to the hosts directly
-#This is OK for the pis etc. but probably not for the control computer
-docker run huntsmanarray/device_startup:latest --network host -it
+#This is OK for the pis etc. but probably not for the control computer.
+
+#-it allows the container to run in interactive mode.
+
+#--cap-add SYS_ADMIN --device /dev/fuse is necessary for the SSHFS mount.
+
+#-v ~/.ssh:/root/.ssh allows the container to use the credentials of the host
+#to setup ssh connections (i.e. the SSHFS). This is used for passwordless
+#login to the control computer, ***which must already be setup between the host
+#device and the control computer***.
+
+#docker run -it --network host --cap-add SYS_ADMIN --device /dev/fuse \
+#        -v ~/.ssh:/root/.ssh:ro huntsmanarray/device_startup:latest 
+#^Unable to set permissions properly this way 
+
+#Create a container 
+CONTAINER_ID=$(docker create -it --network host --cap-add SYS_ADMIN \
+        --device /dev/fuse huntsmanarray/device_startup:latest /bin/bash)
+
+#Start the container
+docker start $CONTAINER_ID
+
+#Extract some environment variables from the container
+CONTAINER_USER=$(docker exec $CONTAINER_ID echo $USER)
+CONTAINER_HOME=$(docker exec $CONTAINER_ID echo $HOME)
+CONTAINER_PANDIR=$(docker exec $CONTAINER_ID echo $PANDIR)
```

Comment:
I'm not sure this is doing the right thing. Those variables after echo will probably be expanded before-hand and they all exist on the host. Have you double-checked this?

Reply:
Excellent point and very well noticed - I think you're right. I'll change it before merging.